CHANGELOG.md

```diff
@@ -5,9 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Changed
+* update dependencies in package,json
+* changed color values of various css elements to enhance accessibility
+* recolor of navbar-inverse
+
 ### Added
-- new class for danger alert with brighter background color
-- new class for success alert with brighter background color
-- new class for displaying inactive table rows
-- several new classes to display a collapsiblesr    
-- examples for the new classes in `index.html`
\ No newline at end of file
+* new class for danger alert with brighter background color
+* new class for success alert with brighter background color
+* new class for displaying inactive table rows
+* several new classes to display a collapsiblesr    
+* examples for the new classes in `index.html`
```

Comment:
Why are you using `*` instead of `-`? The usual syntax uses `-`. (See: https://keepachangelog.com/en/1.0.0/)